netbox_routing/forms/bulk_import/ospf.py

```diff
@@ -44,14 +44,14 @@ class Meta:
 class OSPFInterfaceImportForm(NetBoxModelImportForm):
     instance = CSVModelChoiceField(
         queryset=OSPFInstance.objects.all(),
-        required=False,
-        to_field_name='name',
-        help_text=_('Name of OSPF Instance')
+        required=True,
+        to_field_name='id',
+        help_text=_('ID of OSPF Instance')
```

Comment:
Not sure why we are taking a step back here.  ID would definitely be a step back.

I think it is likely to do with the fact that currently instance name is not unique.  It would likely make more sense to make instance name unique.